pkgs/shells/zsh/grml-zsh-config/default.nix

```diff
@@ -36,7 +36,7 @@ stdenv.mkDerivation rec {
   meta = with lib; {
     description = "grml's zsh setup";
     homepage = "https://grml.org/zsh/";
-    license = licenses.gpl2;
+    license = licenses.gpl2Plus;
```

Comment:
Both `$out/etc/zsh/keephack` and `$out/etc/zsh/zshrc` look gpl2Only:

https://github.com/grml/grml-etc-core/blob/bdc51f9788c98839cfcb248b9a16d87c8f99c3e2/etc/zsh/zshrc#L5

Reply:
This does mention gpl2Plus, so I reckon we need both https://github.com/grml/grml-etc-core/blob/bdc51f9788c98839cfcb248b9a16d87c8f99c3e2/etc/grml/screen_multisession#L5